docs/spec/sources.md

```diff
@@ -299,7 +299,7 @@ The `spec` field is expected to have the following minimum shape:
 type SourceSpec struct {
     // Sink is a reference to an object that will resolve to a domain name or a
```

Comment:
I think you should update the comment also. Simply because it's no longer a domain name, but merely a URI?

Reply:
that is matching the exact comments from the actual used type, see: 
* https://github.com/knative/pkg/blob/master/apis/duck/v1/source_types.go#L51-L53

How about, we update that, to something like:

```
     // Sink is a reference to an object that will resolve to a uri to use as the sink.
```

Reply:
/cc @n3wscott 

happy to apply the change to `pkg` as well

Reply:
+1